sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -136,8 +136,15 @@ trait WindowExecBase extends UnaryExecNode {
           val frame = spec.frameSpecification.asInstanceOf[SpecifiedWindowFrame]
           function match {
             case AggregateExpression(f, _, _, _, _) => collect("AGGREGATE", frame, e, f)
+            case f: OffsetWindowSpec if f.isRelative =>
+              collect("RELATIVE_OFFSET", frame, e, f)
+            case f: OffsetWindowSpec if !f.ignoreNulls &&
+              frame.frameType == RowFrame && frame.lower == UnboundedPreceding =>
+              frame.upper match {
+                case UnboundedFollowing => collect("UNBOUNDED_OFFSET", f.fakeFrame, e, f)
```

Comment:
`UNBOUNDED_FOLLOWING_OFFSET` ?

Reply:
I use the name `UNBOUNDED_OFFSET` because there is unbounded before and after.